-b feature/header

LJJordan124 opened the pull request:
Added Toggle for Dark & Light and organized the appearance of the header

LJJordan124 committed:
-b feature/header

Added Toggle for Dark & Light and organized the appearance of the header

erikbenedict reviewed (changes requested):
This branch is very far behind main. You need to 'git checkout main' then 'git pull'. Then 'git checkout 'your-branch-name-here', then 'git pull origin main'. After this you will need to resolve any merge conflicts that are there. I suggest using the 'Resolve in Merge Editor' tool that is at the bottom of the merge conflict tab in VS Code.

LJJordan124 committed:
-b feature/header

Added Toggle for Dark & Light and organized the appearance of the header

Matched it so it can be properly merged into the main

LJJordan124 committed:
-m header/dark-toggle/hero

Proper Commit Push.
Dark-Toggle has been enabled and effects multiple color changes.
Header looks clean.
Hero Bar has been colored and shaped.

LJJordan124 commented:
Old content. Attempt to fix my commit issues as well.
